nautobot/utilities/tests/test_filters.py

```diff
@@ -1061,28 +1136,43 @@ def get_filterset_count(self, params, klass=None):
     def test_default_icontains(self):
         """Test a default search for an "icontains" value."""
         params = {"q": "Test Site"}
-        self.assertEqual(self.get_filterset_count(params), 4)
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset_class(params, self.queryset).qs, self.queryset.filter(name__icontains="Test Site")
+        )
         params = {"q": "Test Site 3"}
-        self.assertEqual(self.get_filterset_count(params), 1)
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset_class(params, self.queryset).qs, self.queryset.filter(name__icontains="Test Site 3")
+        )
         # Trailing space should only match the first 3.
         params = {"q": "Test Site "}
-        self.assertEqual(self.get_filterset_count(params), 3)
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset_class(params, self.queryset).qs, self.queryset.filter(name__icontains="Test Site ")
+        )
 
     def test_default_exact(self):
         """Test a default search for an "exact" value."""
         params = {"q": "1234"}
-        self.assertEqual(self.get_filterset_count(params), 1)
-        params = {"q": "123"}
-        self.assertEqual(self.get_filterset_count(params), 0)
+        self.assertQuerysetEqualAndNotEmpty(
+            self.filterset_class(params, self.queryset).qs, self.queryset.filter(asn__exact="1234")
+        )
+        asn = Site.objects.values_list("asn", flat=True).first()
```

Comment:
So it doesn't repeat the previous test?
```suggestion
        asn = Site.objects.exclude(asn="1234").values_list("asn", flat=True).first()
```